src/app/modules/spcp/spcp.service.ts

```diff
@@ -409,7 +449,9 @@ export class SpcpService {
     VerifyJwtError | InvalidJwtError | MissingJwtError
   > {
     return this.extractJwt(cookies, authType).asyncAndThen((jwtResult) =>
-      this.extractJwtPayload(jwtResult, authType),
+      authType === AuthType.SP
```

Comment:
prefer explicit switch checks, in case we add more authtypes to the params. At least the compiler will warn us then